src/test/java/com/sap/oss/phosphor/fosstars/data/artifact/ReleaseInfoFromMavenTest.java

```diff
@@ -47,13 +52,17 @@ public void testIfMavenArtifactExist() throws IOException {
 
       provider.update(MAVEN_ARTIFACT, values);
 
-      assertEquals(1, values.size());
+      assertEquals(2, values.size());
       assertTrue(values.has(RELEASED_ARTIFACT_VERSIONS));
       assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).isPresent());
       assertFalse(values.of(RELEASED_ARTIFACT_VERSIONS).get().isUnknown());
       assertFalse(values.of(RELEASED_ARTIFACT_VERSIONS).get().get().empty());
-      assertEquals(15, values.of(RELEASED_ARTIFACT_VERSIONS).get().get().size());
+      assertEquals(20, values.of(RELEASED_ARTIFACT_VERSIONS).get().get().size());
       assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).get().get().get("1.10.10").isPresent());
+      assertFalse(values.of(ARTIFACT_VERSION).get().isUnknown());
+      ArtifactVersion foundArtifactVersion = values.of(ARTIFACT_VERSION).get().get();
+      assertEquals("1.10.10", foundArtifactVersion.version());
+      assertEquals(asLocalDateTime(1618200022000L), foundArtifactVersion.releaseDate());
```

Comment:
Yes, I just copied it from the JSON.
However, if I change it like you proposed we can also get rid of the `asLocalDateTime` which make it even more readable 🙂

Reply:
Unfortunately I run into some issues with timezones 😕
I have for now reverted the change back and will have a look into it a bit later (as otherwise the actions are failing).

Reply:
Sure, no problem.